### PR TITLE
Hotfix add more info to BE wildard string

### PR DESCRIPTION
**Pull request by zonky2**
Hotfix add more info to BE wildard string

see https://github.com/MetaModels/core/issues/269

shot358

### CODE REVIEW

File: src/MetaModels/FrontendIntegration/MetaModelHybrid.php [hunk 1]
```diff
@@ -96,11 +97,41 @@ public function __construct($objElement, $strColumn = 'main')
...
     {
         if (TL_MODE == 'BE') {
+
+            $strInfo = '';
+            if ($this->metamodel) {
+                // Retrieve Name of MetaModels
+                $factory       = $this->getServiceContainer()->getFactory();
+                $metaModelName = $factory->translateIdToMetaModelName($this->metamodel);
+                $metaModel     = $factory->getMetaModel($metaModelName);
+                $strInfo       = '<br><span style="width:31px;display:inline-block;">MM: </span>' . $metaModel->getName();
```
Comment (dmolineus) [inline src/MetaModels/FrontendIntegration/MetaModelHybrid.php]: Styles should be moved to an extra css file.

Comment (dmolineus) [reply src/MetaModels/FrontendIntegration/MetaModelHybrid.php]: Please use the `abbr` tag and provide a title: 

```html
<abbr title="Metamodel">MM</abbr>
```

File: src/MetaModels/FrontendIntegration/MetaModelHybrid.php [hunk 2]
```diff
@@ -96,11 +97,41 @@ public function __construct($objElement, $strColumn = 'main')
...
+
+                $database = \Database::getInstance();
+
+                // Retrieve Name of Filter.
+                if ($this->metamodel_filtering) {
+                    $infoFi = $database
+                        ->prepare('SELECT * FROM tl_metamodel_filter WHERE id=?')
+                        ->execute($this->metamodel_filtering)
+                        ->row();
+                    $strInfo .= '<br><span style="width:31px;display:inline-block;">Fi: </span>' . $infoFi['name'];
```
Comment (dmolineus) [inline src/MetaModels/FrontendIntegration/MetaModelHybrid.php]: Same as above:
- Styles should be moved to an extra css file.
- Please use the `abbr` tag and provide a title.

File: src/MetaModels/FrontendIntegration/MetaModelHybrid.php [hunk 3]
```diff
@@ -96,11 +97,41 @@ public function __construct($objElement, $strColumn = 'main')
...
+                }
+
+                // Retrieve Name of Rendersetting.
+                if ($this->metamodel_rendersettings) {
+                    $infoRs = $database
+                        ->prepare('SELECT * FROM tl_metamodel_rendersettings WHERE id=?')
+                        ->execute($this->metamodel_rendersettings)
+                        ->row();
+                    $strInfo .= '<br><span style="width:31px;display:inline-block;">Rs: </span>' . $infoRs['name'];
```
Comment (dmolineus) [inline src/MetaModels/FrontendIntegration/MetaModelHybrid.php]: Same as above:
- Styles should be moved to an extra css file.
- Please use the `abbr` tag and provide a title.

**Comment by zonky2** [comment]
@dmolineus I add your improvements...

**Comment by dmolineus** [comment]
I'd like to see some more improvements. I'll merge it and add it on my own:
 
 - Use language keys for the labels
 - Code formatting

**Pull request merged**